workitem/workitem_repository_blackbox_test.go

```diff
@@ -322,3 +322,47 @@ func (s *workItemRepoBlackBoxTest) TestLookupIDByNamedSpaceAndNumberNotFound() {
 	require.NotNil(s.T(), err)
 	assert.IsType(s.T(), errors.NotFoundError{}, errs.Cause(err))
 }
+
+func (s *workItemRepoBlackBoxTest) TestLookupIDByNamedSpaceAndNumberStaleSpace() {
+	testFxt := tf.NewTestFixture(s.T(), s.DB, tf.Spaces(1), tf.WorkItems(20, func(testf *tf.TestFixture, idx int) error {
+		testf.WorkItems[idx].Type = workitem.SystemBug
```

Comment:
Please remove this line. The work item will automatically have a valid work item type. The`workitem.SystemBug` is technically not available in every space. It is from the system space and we should use it only there. I know that we currently do otherwise in our code but we should really use the WIT that is generated for you and automatically set for the workitem already.

Reply:
Fixed in https://github.com/fabric8-services/fabric8-wit/pull/1643/commits/5f826703ded88dc2c908170876d5982e276daece